src/Microsoft.DotNet.ImageBuilder/src/Commands/CleanAcrImagesOptions.cs

```diff
@@ -45,5 +53,38 @@ public override void DefineParameters(ArgumentSyntax syntax)
             syntax.DefineParameter("registry", ref registryName, "Name of the registry");
             RegistryName = registryName;
         }
+
+        public override void DefineOptions(ArgumentSyntax syntax)
+        {
+            base.DefineOptions(syntax);
+
+            const CleanAcrImagesAction defaultAction = CleanAcrImagesAction.PruneDangling;
+            string action = defaultAction.ToString().ToCamelCase();
+            syntax.DefineOption("action", ref action,
+                $"Type of delete action. {EnumHelper.GetHelpTextOptions(defaultAction)}");
+            Action = (CleanAcrImagesAction)Enum.Parse(typeof(CleanAcrImagesAction), action, ignoreCase: true);
+
+            int daysOld = 30;
+            syntax.DefineOption("days", ref action, "Number of days old to be considered for deletion");
```

Comment:
The wording of the description was a bit awkward to me on first read.  What do you think about using "age" nomenclature instead?

```suggestion
            syntax.DefineOption("age", ref age, "Minumum age (days) of images to apply delete action to");
```

Note: the original code has the wrong `ref` variable passed - `action`